alicloud/data_source_alicloud_security_group_rules.go

```diff
@@ -0,0 +1,182 @@
+package alicloud
+
+import (
+	"fmt"
+	"log"
+	"strings"
+
+	"github.com/denverdino/aliyungo/ecs"
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func dataSourceAlicloudSecurityGroupRules() *schema.Resource {
+	return &schema.Resource{
+		Read: dataSourceAlicloudSecurityGroupRulesRead,
+
+		Schema: map[string]*schema.Schema{
+			"id": {
```

Comment:
I think 'group_id' may be more easily understandable than 'id'. 'name' and 'description' are also same.  After all, the datasource is security group rules, not security group.

Reply:
Indeed, fixed. Please find update.